custom_components/geappliances/ha_compatibility/data_source.py

```diff
@@ -211,3 +254,12 @@ async def get_entity_id_for_unique_id(
             )
 
         return None
+
+    async def get_erd_status_pair(self, erd: Erd) -> dict[str, str] | None:
+        """Return the status/request pair dict if the given ERD is part of a status/request pair, otherwise None."""
+        for pair in self._status_pair_dict.values():
+            if erd == int(pair["status"], base=16) or erd == int(
+                pair["request"], base=16
+            ):
```

Comment:
```suggestion
            if erd == int(pair["status"], base=16) or erd == int(pair["request"], base=16):
```

Is the formatter forcing this monstrosity on you?